jme3-plugins-json/src/main/java/com/jme3/plugins/json/JsonElement.java

```diff
@@ -99,4 +99,15 @@ public interface JsonElement {
      */
     public <T extends JsonElement> T autoCast();
 
+    /* (non-Javadoc)
+     * @see java.lang.Object#equals(java.lang.Object) 
+     */
+    @Override
+    public boolean equals(Object obj);
+
+    /*(non-Javadoc)
+     * @see java.lang.Object#hashCode() 
+     */
+    @Override
+    public int hashCode();
```

Comment:
Hmm, I don't understand the reason declaring equals and hashCode in a interface? Everything that actually implements this interface is inherently a Object that has these methods already.

Reply:
thanks for the observation, fixed...